WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/creation/totals/OrderCreateEditTotalsView.kt

```diff
@@ -1,67 +1,214 @@
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.woocommerce.android.R
 import com.woocommerce.android.ui.compose.component.WCColoredButton
-import com.woocommerce.android.ui.compose.theme.WooThemeWithBackground
+import com.woocommerce.android.ui.compose.theme.WooTheme
+import kotlinx.coroutines.launch
 
 @Composable
-fun OrderCreateEditTotalsView(state: TotalsSectionsState.Shown) {
-    WooThemeWithBackground {
-        Card(
-            shape = RoundedCornerShape(0.dp),
-            elevation = 0.dp
+fun OrderCreateEditTotalsView(
+    state: TotalsSectionsState,
+    isPreview: Boolean = LocalInspectionMode.current,
+) {
+    WooTheme {
+        val visible = state is TotalsSectionsState.Shown
+        AnimatedVisibility(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight(),
+            visible = visible || isPreview,
+            enter = fadeIn() + slideInVertically(initialOffsetY = { it }),
+            exit = slideOutVertically(targetOffsetY = { it }) + fadeOut(),
         ) {
             Column(
                 modifier = Modifier
                     .fillMaxWidth()
-                    .wrapContentHeight(),
-                horizontalAlignment = Alignment.CenterHorizontally
+                    .wrapContentHeight()
+                    .background(color = Color.Transparent),
             ) {
-                Divider(modifier = Modifier.fillMaxWidth())
-
-                Icon(
-                    imageVector = Icons.Default.KeyboardArrowUp,
-                    contentDescription = null,
-                    tint = colorResource(id = R.color.color_primary)
+                val shadowHeight = dimensionResource(id = R.dimen.minor_100)
+                val shadowHeightPx = with(LocalDensity.current) { shadowHeight.toPx() }
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .background(
+                            brush = Brush.verticalGradient(
+                                colors = listOf(
+                                    Color.Transparent,
+                                    Color.Black.copy(alpha = 0.15f),
+                                ),
+                                startY = 0f,
+                                endY = shadowHeightPx,
+                            )
+                        )
+                        .height(shadowHeight)
                 )
 
-                Spacer(modifier = Modifier.height(24.dp))
+                TotalsView(state)
+            }
+        }
+    }
+}
 
-                WCColoredButton(
-                    onClick = { state.button.onClick },
-                    modifier = Modifier
-                        .fillMaxWidth()
-                        .padding(horizontal = dimensionResource(id = R.dimen.major_100)),
+@Composable
+private fun TotalsView(
+    state: TotalsSectionsState,
```

Comment:
```suggestion
    state: TotalsSectionsState.Shown,
```
Can we pass `TotalsSectionsState.Shown` here? I guess we don't show this component at all if it's `Hidden`.

Reply:
Yep, that'll done in the following PR

Reply:
Furthermore, it looks like that we decided to go with always visible drawer so `Hidden` in the following PRs